src/analysis/mean_inequalities.lean

```diff
@@ -292,6 +292,20 @@ theorem young_inequality (a b : ℝ≥0) {p q : ℝ≥0} (hp : 1 < p) (hpq : 1 /
   a * b ≤ a^(p:ℝ) / p + b^(q:ℝ) / q :=
 real.young_inequality_of_nonneg a.coe_nonneg b.coe_nonneg ⟨hp, nnreal.coe_eq.2 hpq⟩
 
+/-- Young's inequality, `ℝ≥0` version with real conjugate exponents. -/
+theorem young_inequality_real (a b : ℝ≥0) {p q : ℝ} (hpq : p.is_conjugate_exponent q) :
+  a * b ≤ a ^ p / nnreal.of_real p + b ^ q / nnreal.of_real q :=
+begin
+  have hp : 1 < nnreal.of_real p,
+  { rw [←of_real_one, of_real_lt_of_real_iff hpq.pos], exact hpq.one_lt, },
+  have hpq_nnreal : 1 / nnreal.of_real p + 1 / nnreal.of_real q = 1,
+  { rw [←of_real_one, ←of_real_div' hpq.nonneg, ←of_real_div' hpq.symm.nonneg,
+      ←of_real_add hpq.one_div_nonneg hpq.symm.one_div_nonneg, hpq.inv_add_inv_conj], },
```

Comment:
Maybe you could make two lemmas out of these statements, in the `is_conjugate_exponent` file (it is designed precisely to record this kind of triviality, to make it easily available through the dot notation)